### PR TITLE
Wildcards exports

### PR DESCRIPTION
**Pull request by grod220**
Closes https://github.com/penumbra-zone/web/issues/750

### CODE REVIEW

File: packages/client/package.json [hunk 1]
```diff
@@ -3,14 +3,20 @@
...
-  ],
   "exports": {
-    ".": "./dist/prax.js",
-    "./global": "./dist/global.js",
-    "./src/global": "./dist/global.js",
-    "./get-port": "./dist/get-port.js"
+    ".": "./src/prax.ts",
+    "./src/global": "./src/global.ts",
+    "./src/get-port": "./src/get-port.ts"
+  },
```
Comment (grod220) [inline packages/client/package.json:10]: Note: this re-introduces internal packages (if used internally)

File: turbo.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
       "cache": false,
       "persistent": true
     },
-    "lint": { "dependsOn": ["compile", "^build"] },
+    "lint": { "dependsOn": ["compile"] },
```
Comment (grod220) [inline turbo.json:25]: Now the external packages are being used as internal again, we don't need to pre-build them

**Review by Valentine1898 — APPROVED** [review]
LGTM

File: packages/getters/package.json [hunk 3]
```diff
@@ -3,31 +3,13 @@
...
-    "./src/validator-state": "./dist/validator-state.js",
-    "./src/validator-status": "./dist/validator-status.js",
-    "./src/value-view": "./dist/value-view.js"
+  "publishConfig": {
+    "files": [
+      "dist"
+    ],
+    "exports": {
+      "./src/*": "./dist/*.js"
+    }
```
Comment (turbocrime) [inline packages/getters/package.json:12]: :D

**Pull request merged**